deployment/deployment.json

```diff
@@ -1545,8 +1545,8 @@
         },
         "services": {
           "hosted-service": {
-            "slug": "venus-protocol-bsc",
-            "query-id": "venus-protocol-bsc"
+            "slug": "venus-bsc",
```

Comment:
I had thought this is causing the build error for venus-bsc: https://github.com/messari/subgraphs/actions/runs/3186399411/jobs/5196948176 .

Looks like it may not be, as the error remains with the renaming. Still looking...

Reply:
I believe it is indeed the cause (there needs one more renaming). 

Not sure what the convention is. Since the directory for the venus protocol inside compound-forks/protocols is called "venus", the slug and query id should be "venus-bsc". It seems inconsistent to name them "venus-protocol-bsc"

Reply:
Since we originally named the hosted service subgraph "messari/venus-protocol-bsc" we just left it.

I don't believe it is what is failing the build.